onnxruntime/core/graph/graph_utils.h

```diff
@@ -80,6 +81,13 @@ bool RemoveNode(Graph& graph, Node& node);
     This should probably be elevated to the Graph API eventually. */
 size_t RemoveNodeOutputEdges(Graph& graph, Node& node);
 
+/** Creates a TensorProto that will be used as an initializer to a node.
+    @param[in] out_tensor the Tensor whose data and shape will be used to create the TensorProto.
+    @param[in] node_arg the NodeArg of the Node for which the TensorProto will be used as an initializer.
+    @param[out] tensor_proto the TensorProto. */
+void BuildTensorProtoForInitializer(const Tensor& out_tensor, const NodeArg& node_arg,
```

Comment:
>out_tensor [](start = 50, length = 10)

Whilst it may be an output tensor at the calling site, 'out_tensor' is a confusing name here as it's the input